mappings/net/minecraft/entity/mob/DrownedEntity.mapping

```diff
@@ -11,6 +11,8 @@ CLASS net/minecraft/class_1551 net/minecraft/entity/mob/DrownedEntity
 		ARG 2 spawnType
 		ARG 3 pos
 		ARG 4 random
+	METHOD method_7012 canDrownedTarget (Lnet/minecraft/class_1309;)Z
```

Comment:
```suggestion
	METHOD method_7012 canAttackTarget (Lnet/minecraft/class_1309;)Z
```

Reply:
This is a method containing only logic specific to drowned, though

Reply:
`canDrownedAttackTarget` then?